README.md

```diff
@@ -25,7 +25,40 @@ kubectl apply -f https://github.com/open-feature/open-feature-operator/releases/
 
 <!---x-release-please-end-->
 
-### Architecture
+### Release contents
+
+- `certificate.yaml` holds the cert-manager manifests used to authorize requests between components
+- `release.yaml` contains the configuration of
+  - `FeatureFlagConfiguration` `CustomResourceDefinition` (custom type that holds the configured state of feature flags)
+  - standard kubernetes primitives (e.g. namespace, accounts, roles, bindings, configmaps)
```

Comment:
Should these bullet points start with uppercases as they are still sentences?

Reply:
Done